src/coreComponents/linearAlgebra/interfaces/hypre/HypreInterface.cpp

```diff
@@ -18,16 +18,27 @@
 
 #include "HypreInterface.hpp"
 #include "linearAlgebra/interfaces/hypre/HyprePreconditioner.hpp"
+#include "HYPRE_utilities.h"
+#include "_hypre_utilities.h"
 
 namespace geosx
 {
 
 void HypreInterface::initialize( int & GEOSX_UNUSED_PARAM( argc ),
                                  char * * & GEOSX_UNUSED_PARAM( argv ) )
-{}
+{
+  HYPRE_Init();
+#if defined(USE_CUDA)
+  HYPRE_ExecutionPolicy default_exec_policy = HYPRE_EXEC_DEVICE;
```

Comment:
Sorry. I didn't grab it as part of the previous comment. This function is called as part of the GEOSX initialization process...so prior to anything else happening.

Reply:
@rrsettgast Just confirming that your are linking against a GPU build of hypre. That is, hypre is configured --with-cuda .

Reply:
yes

Reply:
@rrsettgast Any new updates? Are you able to link to the GPU build correctly now, and are you seeing the kernels with nvprof?